chart/k8gb/values.yaml

```diff
@@ -62,3 +62,5 @@ route53:
 
 ns1:
   enabled: false
+  # endpoint: http://ns1.example.com
```

Comment:
please mentioned in the comment that the value is optional and it defaults to managed NS1

Reply:
@ytsarev done